### PR TITLE
Feature lwdev 5362 models package

### PR DESCRIPTION
**Pull request by mihail-yartsev**

### CODE REVIEW

File: src/MarginTrading.Contract/ClientContracts/TradeContract.cs [hunk 1]
```diff
@@ -1,18 +1,33 @@
...
         public string AssetPairId { get; set; }
+        
+        [DisplayName("Trade type on order's side: buy or sell")]
         [JsonConverter(typeof(StringEnumConverter))]
         public TradeClientType Type { get; set; }
+        
+        [DisplayName("Trade date and time")]
         public DateTime Date { get; set; }
+
+        [DisplayName("Trade effective prive")]
```
Comment (andrey-anokhin) [inline src/MarginTrading.Contract/ClientContracts/TradeContract.cs]: *price

File: src/MarginTrading.Contract/ClientContracts/UserUpdateEntityClientContract.cs [hunk 2]
```diff
@@ -1,8 +1,13 @@
-﻿namespace MarginTrading.Contract.ClientContracts
+﻿using System.ComponentModel;
+
+namespace MarginTrading.Contract.ClientContracts
 {
+    [DisplayName("User update info")]
     public class UserUpdateEntityClientContract
     {
+        [DisplayName("Inicates account assets were updated")]
```
Comment (andrey-anokhin) [inline src/MarginTrading.Contract/ClientContracts/UserUpdateEntityClientContract.cs]: Indicates?

File: src/MarginTrading.Contract/ClientContracts/UserUpdateEntityClientContract.cs [hunk 3]
```diff
@@ -1,8 +1,13 @@
...
         public bool UpdateAccountAssetPairs { get; set; }
+        [DisplayName("Inicates accounts were updated")]
```
Comment (andrey-anokhin) [inline src/MarginTrading.Contract/ClientContracts/UserUpdateEntityClientContract.cs]: Indicates?

Comment (mihail-yartsev) [reply src/MarginTrading.Contract/ClientContracts/UserUpdateEntityClientContract.cs]: fixed

File: src/MarginTrading.Contract/ClientContracts/UserUpdateEntityClientContract.cs (hunk 2 above)
Comment (mihail-yartsev) [reply src/MarginTrading.Contract/ClientContracts/UserUpdateEntityClientContract.cs]: fixed

File: src/MarginTrading.Contract/ClientContracts/TradeContract.cs (hunk 1 above)
Comment (mihail-yartsev) [reply src/MarginTrading.Contract/ClientContracts/TradeContract.cs]: fixed

**Pull request merged**